apps/sensenet/cypress/integration/saved-queries/saved-queries.spec.ts

```diff
@@ -0,0 +1,48 @@
+import { pathWithQueryParams } from '../../../src/services/query-string-builder'
+
+describe('Saved queries', () => {
+  before(() => {
+    cy.login()
+    cy.visit(pathWithQueryParams({ path: '/', newParams: { repoUrl: Cypress.env('repoUrl') } }))
+  })
+  it('saving a query should work properly.', () => {
+    const inputText = "+InFolder:'/Root/Content/IT/Document_Library'"
+    cy.get('[data-test="drawer-menu-item-Search"]').click()
+    cy.get('a[title="New search"]').click()
+    cy.get('[data-test="input-search"]').type(inputText)
+    cy.get('[data-test="table-cell-IT Workspace"]').should('be.visible')
+    cy.get('[data-test="table-cell-Queries"]').should('be.visible')
+    cy.get('[data-test="table-cell-Sample workspace"]').should('be.visible')
```

Comment:
Here you should check if Chicago and Munich are in the list
```suggestion
    cy.get('[data-test="table-cell-Chicago"]').should('be.visible')
    cy.get('[data-test="table-cell-Munich"]').should('be.visible')
```

Reply:
@pusztaienike changes done.